server/api/users.go

```diff
@@ -278,7 +289,7 @@ func (a *API) handleGetUser(w http.ResponseWriter, r *http.Request) {
 	vars := mux.Vars(r)
 	userID := vars["userID"]
 
-	auditRec := a.makeAuditRecord(r, "postBlocks", audit.Fail)
+	auditRec := a.makeAuditRecord(r, "getUser", audit.Fail)
```

Comment:
why?

Reply:
I noticed that the event name described the wrong event (you can see "postBlocks" as the event name in function handlePostBlocks(), so I suppose this was a copy-paste), so I fixed it. 

This has nothing to do with the password change)